tests/python_tests/test_whisper_generate_api.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (C) 2023-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import openvino_genai as ov_genai
+import functools
+import pytest
+import numpy as np
+import openvino_tokenizers
+import openvino
+from ov_genai_test_utils import get_whisper_models_list
+import pathlib
+import os
+import requests
+from urllib.parse import urlparse
+from scipy.io import wavfile
+
+
+@functools.lru_cache(1)
+def read_whisper_model(params, **tokenizer_kwargs):
+    model_id, path = params
+
+    from optimum.intel.openvino import OVModelForSpeechSeq2Seq
+    from transformers import AutoTokenizer
+
+    tokenizer = AutoTokenizer.from_pretrained(model_id, trust_remote_code=True)
+
+    if (path / "openvino_encoder_model.xml").exists():
+        opt_model = OVModelForSpeechSeq2Seq.from_pretrained(
+            path, trust_remote_code=True, compile=False, device="CPU"
+        )
+    else:
+        ov_tokenizer, ov_detokenizer = openvino_tokenizers.convert_tokenizer(
+            tokenizer, with_detokenizer=True, **tokenizer_kwargs
+        )
+        openvino.save_model(ov_tokenizer, path / "openvino_tokenizer.xml")
```

Comment:
looks like we don't need tokenizer for whipser? only detokenizer

Reply:
Whisper doesn't need tokenizer so we could removed. The problem is that `ov::genai::Tokenizer` initializes both tokenizer and detokenizer in constructor. We could improve it by initializing only detokenizer for Whisper pipeline, but I propose to do it in a separate PR. I'll add todo for that.